docs/concepts/fundamentals/slots.md

```diff
@@ -663,3 +663,42 @@ class MyTable(Component):
     </div>
     """
 ```
+
+### Escaping slots content
```

Comment:
This bit about escaping slots was in the "Componens as views" page. I moved it here instead, and I've mentioned it in "Getting started" (`docs/getting_started/rendering_components.md`) too.